src/webapp/components/replicate/ReplicateUserFromTable.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useEffect, useMemo } from "react";
+import i18n from "../../../locales";
+import { Id } from "../../../domain/entities/Ref";
+import { useAppContext } from "../../contexts/app-context";
+import { ImportTable, Columns } from "../import-export/ImportTable";
+import { User, defaultUser } from "../../../domain/entities/User";
+import { useLoading, useSnackbar } from "@eyeseetea/d2-ui-components";
+import { generateUid } from "../../../utils/uid";
+
+interface ReplicateUserFromTableProps {
+    userToReplicateId: Id;
+    onRequestClose: () => void;
+}
+
+export const ReplicateUserFromTable: React.FC<ReplicateUserFromTableProps> = props => {
+    const { compositionRoot } = useAppContext();
```

Comment:
The component should only render. (render logic)

The rest of the presentation logic (manage state, actions, invoke use cases, etc.) should be in a custom hook: useReplicateUserFromTable.

In this custom hook, we should use the user entity, not the user props